### PR TITLE
Rubocop --auto-correct syntax fixups

### PR DESCRIPTION
**Pull request by sschneid**

### CODE REVIEW

File: lib/vmpooler/api.rb [hunk 1]
```diff
@@ -8,25 +8,24 @@ def initialize
...
       $config[:redis]['server'] ||= 'localhost'
 
-      if ($config[:graphite]['server'])
+      if $config[:graphite]['server']
         $config[:graphite]['prefix'] ||= 'vmpooler'
       end
 
       # Connect to Redis
-      $redis = Redis.new(:host => $config[:redis]['server'])
+      $redis = Redis.new(host: $config[:redis]['server'])
```
Comment (colinPL) [inline lib/vmpooler/api.rb]: Is `host` still a symbol here?

**Comment by rji** [comment]
Seems legit. :+1:

File: lib/vmpooler/api.rb (hunk 1 above)
Comment (sschneid) [inline lib/vmpooler/api.rb]: I guess?  I deployed the reformatted code to dev and everything seems to function identically AFAICT.

File: lib/vmpooler/api.rb [hunk 2]
```diff
@@ -292,12 +291,12 @@ def hostname_shorten hostname
...
 
           request.keys.each do |template|
-            if ( $redis.scard('vmpooler__ready__'+template) < request[template] )
+            if $redis.scard('vmpooler__ready__' + template) < request[template]
               available = 0
             end
           end
 
-          if ( available == 1 )
+          if (available == 1)
```
Comment (colinPL) [inline lib/vmpooler/api.rb]: Why did it keep the parens here?

**Comment by colinPL** [comment]
:+1:

**Pull request merged**